module/x/peggy/client/rest/query.go

```diff
@@ -134,3 +134,24 @@ func lastValsetRequestsByAddressHandler(cliCtx context.CLIContext, storeName str
 		rest.PostProcessResponse(w, cliCtx.WithHeight(height), res)
 	}
 }
+
+func lastObservedNonceHandler(cliCtx context.CLIContext, storeName string) func(http.ResponseWriter, *http.Request) {
```

Comment:
can we make it more clear which of the two contract nonces this is?

Reply:
This endpoint supports all nonces for the claim types we have : [bridge_deposit bridge_withdrawal_batch bridge_multisig_update]